### PR TITLE
Pagination fix

### PR DESCRIPTION
**Pull request by peterMuriuki**
Pagination issue:
select a pagination that is in the far end.
add a filter so that the number pagination items reduces past the selected pagination item,
not the pagination breaks

Replaces the custom pagination implementation with a library
Refactor the tests

### CODE REVIEW

File: src/components/VillageData/index.tsx [hunk 1]
```diff
@@ -42,19 +42,30 @@ class VillageData extends React.Component<VillageDataPropsType, State> {
...
-            startLabel: t('first'),
-            totalRecords: this.props.smsData.length,
+            containerClassName: 'pagination',
+            activeClassName: 'active',
+            pageClassName: 'page-item',
+            previousClassName: 'page-item',
+            nextClassName: 'page-item',
+            pageLinkClassName: 'page-link',
+            previousLinkClassName: 'page-link',
+            nextLinkClassName: 'page-link',
```
Comment (moshthepitt) [inline src/components/VillageData/index.tsx]: Is making this more DRY something we want to pursue?

File: src/containers/LogFace/tests/__snapshots__/index.test.tsx.snap [hunk 2]
```diff
@@ -342,284 +342,213 @@ exports[`containers/LogFace renders correctly: logface title 1`] = `
 `;
```
Comment (moshthepitt) [inline src/containers/LogFace/tests/__snapshots__/index.test.tsx.snap]: Is this snapshot too large?

File: src/components/VillageData/index.tsx (hunk 1 above)
Comment (peterMuriuki) [reply src/components/VillageData/index.tsx]: moved this to util function

**Comment by peterMuriuki** [comment]
coverage drop due to removed code

**Review by moshthepitt — DISMISSED** [review]

File: src/components/VillageData/index.tsx [hunk 3]
```diff
@@ -42,19 +42,17 @@ class VillageData extends React.Component<VillageDataPropsType, State> {
     public render() {
         const { t } = this.props;
 
-        const routePaginatorProps: PaginatorProps = {
-            endLabel: t('last'),
-            nextLabel: t('next'),
-            onPageChange: (paginationData: PaginationData) => {
+        const pageLimit = 5;
```
Comment (kahummer) [inline src/components/VillageData/index.tsx]: I think we should move this to constants

File: src/containers/LogFace/index.tsx [hunk 4]
```diff
@@ -157,19 +158,16 @@ const LogFace = (props: LogFacePropsType) => {
         updateUrlWithFilter(SEARCH_FILTER_PARAM, props, (event.target as HTMLInputElement).value);
     };
 
-    const onPageChangeHandler = (paginationData: PaginationData) => {
-        setCurrentPage(paginationData.currentPage);
+    const onPageChangeHandler = (page: any) => {
```
Comment (kahummer) [inline src/containers/LogFace/index.tsx]: why do we have any?

**Review by kahummer — DISMISSED** [review]

**Review by kahummer — APPROVED** [review]

**Review by moshthepitt — APPROVED** [review]

**Pull request merged**